src/CONST.ts

```diff
@@ -3565,26 +3564,23 @@ const CONST = {
             '\n' +
             "We'll store all expenses in your new workspace for easy access. Let me know if you have any questions!",
         [onboardingChoices.EMPLOYER]:
-            '# Welcome to Expensify, the fastest way to get paid back!\n' +
-            "Hi there, I'm Concierge. Chat with me here for anything you need.\n" +
+            '# Expensify is the fastest way to get paid back!\n' +
             '\n' +
             'To submit expenses for reimbursement:\n' +
             '1. From the home screen, click the green + button > Request money.\n' +
             "2. Enter an amount or scan a receipt, then input your boss's email.\n" +
             '\n' +
             "That'll send a request to get you paid back. Let me know if you have any questions!",
         [onboardingChoices.MANAGE_TEAM]:
-            "# Welcome to Expensify, let's start managing your team's expenses!\n" +
-            "Hi there, I'm Concierge. Chat with me here for anything you need.\n" +
+            "# Let's start managing your team's expenses!\n" +
             '\n' +
             "To manage your team's expenses, create a workspace to keep everything in one place. Here's how:\n" +
             '1. From the home screen, click the green + button > New Workspace\n' +
             '2. Give your workspace a name (e.g. “Sales team expenses”).\n' +
```

Comment:
@danielrvidal @MariaHCD  There are some inconsistencies in the double quotation marks. Here, we are using the Left Double Quotation Mark and the Right Double Quotation Mark in “Sales team expenses”, IMO, it should just use the simple quotation mark like other CONSTs.

Reply:
For:  **Track and budget personal spend**

```'2. Give your workspace a name (e.g. "My expenses”).\n' +```

It should have used same quotation marks in **"My expenses”**

Reply:
I agree, let's use the simple quotation marks in all the copy here 👍🏼